ui/src/components/CubePreview.vue

```diff
@@ -181,12 +183,11 @@ export default class extends Vue {
     return ''
   }
 
-  get warningMessage (): string | null {
-    if (this.dimensions.length > 0 && !this.dimensions.some(({ isMeasureDimension }) => isMeasureDimension)) {
-      return 'No Measure dimension defined'
-    }
-
-    return null
+  get errors (): Schema.Thing[] {
+    return [
+      ...this.cubeMetadata.errors!,
+      ...this.dimensionMetadataCollection.errors!,
```

Comment:
Combine all errors from cube dataset and metadata collection. This way we can place them freely depending on where they belong. The issues #949 and #948 will probably add to the former